common/metrics/defs.go

```diff
@@ -1143,6 +1143,7 @@ const (
 	PersistenceErrCurrentWorkflowConditionFailedCounter
 	PersistenceErrTimeoutCounter
 	PersistenceErrBusyCounter
+	PersistenceErrTransactionSizeCounter
```

Comment:
I think you can get rid of this counter as we already emit the transaction size from application layer.